BREAKING-CHANGES.md

```diff
@@ -136,8 +156,6 @@ This change simplifies the trait and makes it easier to implement.
 
 ### `Frame::size` is deprecated and renamed to `Frame::area`
 
-[#1293]: https://github.com/ratatui/ratatui/pull/1293
-
```

Comment:
Is there a reason this was removed?

Reply:
This was unintentional, and done automatically as it was unused (It's not referenced in the line above, this is an error)

Reply:
bd5fa963f..eaa180cc6